events/events.c

```diff
@@ -187,6 +198,13 @@ events_run(void)
 {
 	int rc;
 
+	/* Register the atexit function (if necessary). */
+	if (!registered_atexit && atexit(events_atexit)) {
+		warn0("atexit(events_atexit)");
```

Comment:
I'm not 100% certain that we want to add a dependency on `warnp.h` to the event layer.